mailpoet/assets/js/src/automation/editor/components/icons/colored-icon.tsx

```diff
@@ -0,0 +1,26 @@
+import { Icon } from '@wordpress/icons';
+import { IconProps } from '@wordpress/icons/build-types/icon';
+
+export type ColoredIconProps = {
+  width: string;
+  height: string;
+  color: string;
+} & IconProps;
+
+export function ColoredIcon(props: ColoredIconProps): JSX.Element {
+  return (
+    <div
+      className="colored-icon"
+      style={{
+        width: props.width,
+        height: props.height,
+      }}
+    >
+      <div
+        className="colored-icon-background"
+        style={{ backgroundColor: props.color }}
+      />
+      <Icon {...props} />
+    </div>
+  );
+}
```

Comment:
I'll leave the comment here, but it may belong rather to the modified SVGs or the CSS of this component, I'm not sure. The issue is the icons seem to have wrong sizes now:

In the workflow, the `email` icon seems to be too small, compared to Figma, and the `delay` icon is somehow broken:
<img width="183" alt="Screen Shot 2022-07-29 at 9 07 17" src="https://user-images.githubusercontent.com/141436/181702887-86ecff0a-5881-4cb0-96ac-0b7b5693adea.png">

The correct should be:
<img width="100" alt="Screen Shot 2022-07-29 at 9 10 42" src="https://user-images.githubusercontent.com/141436/181703367-95baad94-005c-4933-b10b-5866e7f8b220.png">

Are also the background colors wrong?

In the sidebar, the delay icon seems to be too big now:
<img width="144" alt="Screen Shot 2022-07-29 at 9 07 37" src="https://user-images.githubusercontent.com/141436/181702948-a9d99a21-8477-4434-b11e-76cbe93835c7.png">

And as mentioned, the `delay` icon seems to be somehow damaged, it has even a different shape. Did you try to compress it somehow? Please let's revert such changes and only add necessary fixes for the colors, if possible.

Reply:
Icon switched back and some other improvements done.